### DISCUSSION TITLE
Add Black Bars to Expand Resolution

### DISCUSSION THREAD

**Discussion by Jay123210599** [ImageMagick/ImageMagick#7303] [Help]
### ImageMagick version

7.1.1-27

### Operating system, version and so on

Windows 11

### Description

I have images that are 768x1200, and I want to add black bars to them so that their resolution is 1920x1080. How do I do that?

**Comment by snibgo** [comment]
Your input is taller than your output. Just adding black bars won't do that.

**Reply by fmw42** [reply]
As snibgo says, your input is 1200 px tall but you want your output to be 1080 px tall, which is smaller than the input.  Are you willing to crop the input so that the output can be 1080 in place of 1200 or do you want to shrink the input height using by resizing it, then add black bars to fill the width?

**Reply by Jay123210599** [reply]
Shrink the input.

**Reply by fmw42** [reply]
try

`magick input.suffix -resize "1920x1080^" -gravity center -background black -extent "1920x1080" output.suffix
`

escape the ^ as appropriate for your OS, if needed.

**Reply by fmw42** [reply]
Note: I edited to add -gravity center. See above

**Reply by Jay123210599** [reply]
Didn't work. It only cut out important parts of my image.

**Comment by fmw42** [comment]
Post your input image so I can examine it and then test with it and correct my command

**Reply by Jay123210599** [reply]
Here is it.
![Yume03](https://github.com/ImageMagick/ImageMagick/assets/141977822/527b48a0-3d3d-4acc-abb2-5e8d65bcc850)

**Comment by fmw42** [comment]
try

`magick x.jpg -resize "x1080" -gravity center -background black -extent "1920x1080" y.jpg`
